packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -519,6 +519,49 @@ class PipelineController extends CanvasController {
     return "";
   }
 
+  getPropertyValue(value: any, info?: any, label?: string): any {
+    if (info?.data?.format === "inputpath" || info?.format === "inputpath") {
+      // Find the node corresponding to the input node
+      const upstreamNode = this.findExecutionNode(value?.value ?? "");
+      const upstreamNodeLabel = upstreamNode?.app_data?.ui_data?.label;
+      const upstreamNodeDef = this.getAllPaletteNodes().find(
+        (nodeDef) => nodeDef.op === upstreamNode?.op
+      );
+      // Add each property with a format of outputpath to the options field
+      const upstreamNodeOption = upstreamNodeDef?.app_data?.properties?.uihints?.parameter_info?.find(
+        (prop) => {
+          return prop.parameter_ref === value.option;
+        }
+      )?.label?.default;
+      return {
+        label: label,
+        value:
+          upstreamNodeLabel && upstreamNodeOption
+            ? `${upstreamNodeLabel}: ${upstreamNodeOption ?? ""}`
+            : "No value specified.",
```

Comment:
This change doesn't fix the nit I was referring to, it also breaks the case that the ternary was originally trying to address.

IIUC I think this code block would be correct:

```
upstreamNodeLabel 
   ? upstreamNodeLabel + (upstreamNodeOption ? ": " + upstreamNodeOption : "")
    : "No value specified.",
```

based on how it's written here: https://github.com/elyra-ai/pipeline-editor/blob/cc00ee73297b018a3564503cd343bb8b71f2e903/packages/pipeline-editor/src/CustomFormControls/NestedEnumControl.tsx#L83

Reply:
the three cases are:

1. `Selected Parent Node Name` (we see this in airflow and some kfp)
2. `Select Parent Node Name: selected output of that parent` (we see this in kfp)
3. No value set (default string is used)